string link_name
geometry_msgs/WrenchStamped contact_force
---
bool success
string message
